api/types/authentication.go

```diff
@@ -771,6 +775,16 @@ func (c *AuthPreferenceV2) CheckAndSetDefaults() error {
 		return trace.BadParameter("missing required Webauthn configuration for headless=true")
 	}
 
+	// Prevent accidental local lockout by disabling local second factor methods, (likely leaving only sso enabled).
```

Comment:
Could we get test coverage for this?

Reply:
6ab4fdc5560b7900cb21ab497d5970d3d9fa19c9